package.json

```diff
@@ -49,20 +49,18 @@
     "@primer/blueprints": "4.0.1",
     "@primer/components": "12.0.1",
     "@primer/next-pages": "0.0.3",
-    "@storybook/addon-viewport": "5.0.0",
-    "@storybook/react": "5.0.10",
+    "@storybook/addon-viewport": "5.0.11",
+    "@storybook/react": "5.0.11",
```

Comment:
Did you double check that storybook still runs fine? We just changed the storybook version but I can’t remember if we upgraded or downgraded it to make it work. Either way it would be great to check it’s all working fine if you haven’t already!